cpp/tests/worker.cpp

```diff
@@ -67,6 +82,10 @@ class WorkerProgressTest : public WorkerTest,
   }
 };
 
+class WorkerGenericCallbackTest : public WorkerProgressTest {};
+
+class WorkerGenericCallbackSingleTest : public WorkerProgressTest {};
```

Comment:
It seems that the only form to specify only certain tests should run with a specific parameter set is to have a different derived class to appropriately inform `INSTANTIATE_TEST_SUITE_P`, therefore these classes just inherit the parent to explicitly allow selecting a different set of tests to run.